MAINTAINERS.yml

```
@@ -0,0 +1,1050 @@
+    collaborators:
+        - MaureenHelm
+    files:
+        - drivers/entropy/*rv32m1*
+
+"Drivers: EEPROM":
+    status: maintained
+    maintainers:
+        - henrikbrixandersen
+    files:
+        - drivers/eeprom/
+        - dts/bindings/mtd/*eeprom*
+        - include/drivers/eeprom.h
+        - tests/drivers/eeprom/
+    labels:
+        - "area: EEPROM"
+
+"Drivers: Ethernet":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+        - pfalcon
+    files:
+        - drivers/ethernet/
+    labels:
+        - "area: Ethernet"
+
+"Drivers: Flash":
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - drivers/flash/
+        - dts/bindings/flash_controller/
+        - include/drivers/flash.h
+        - include/flash.h
+        - samples/drivers/flash_shell/
+    labels:
+        - "area: Flash"
+
+"Drivers: GPIO":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/gpio.rst
+        - drivers/gpio/
+        - include/drivers/gpio/
+        - include/drivers/gpio.h
+        - include/gpio.h
+        - include/dt-bindings/gpio/
+        - tests/drivers/gpio/
+        - samples/drivers/gpio/
+    labels:
+        - "area: GPIO"
+
+"Drivers: HW Info":
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    files:
+        - drivers/hwinfo/
+        - dts/bindings/hwinfo/
+        - include/drivers/hwinfo.h
+        - include/hwinfo.h
+        - tests/drivers/hwinfo/
+    labels:
+        - "area: HWINFO"
+
+"Drivers: IEEE 802.15.4":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+    files:
+        - drivers/ieee802154/
+
+"Drivers: Interrupt controllers":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/interrupt_controller/
+        - dts/bindings/interrupt-controller/
+        - include/drivers/interrupt_controller/
+        - include/dt-bindings/interrupt-controller/
+
+"Drivers: IPM":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - drivers/ipm/
+    description: >-
+        Inter-processor mailboxes
+
+"Drivers: I2C":
+    status: orphaned
+    files:
+        - drivers/i2c/
+        - dts/bindings/i2c/
+        - include/i2c.h
+        - include/drivers/i2c.h
+        - include/drivers/i2c/
+    labels:
+        - "area: I2C"
+
+"Drivers: I2S":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - doc/reference/peripherals/i2s.rst
+        - drivers/i2s/
+        - dts/bindings/i2s/
+        - include/i2s.h
+        - include/drivers/i2s.h
+        - tests/drivers/i2s/
+    labels:
+        - "area: I2S"
+
```

```suggestion
"Drivers: I2S stm32":
    # Provisional entry, to be corrected
    collaborators:
        - avisconti
    files:
        - drivers/i2s/i2s_ll_stm32*
```
@avisconti, can you confirm ?

---

yes!